UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Data/Gas.cs

```diff
@@ -9,25 +9,45 @@ public struct Gas
 		// Gas constant
 		public const float R = 8.3144598f;
 
-		public static readonly Gas Plasma = new Gas("Plasma", 200, 40f);
-		public static readonly Gas Oxygen = new Gas("Oxygen", 20, 31.9988f);
-		public static readonly Gas Nitrogen = new Gas("Nitrogen", 20, 28.0134f);
-		public static readonly Gas CarbonDioxide = new Gas("Carbon Dioxide", 30, 44.01f);
+		public static readonly Gas Plasma = new Gas("Plasma", 200, 40f, true, 0.4f, "PlasmaAir", -4);
+		public static readonly Gas Oxygen = new Gas("Oxygen", 20, 31.9988f, false, 0.4f, "NONE", -3);
+		public static readonly Gas Nitrogen = new Gas("Nitrogen", 20, 28.0134f, false, 0.4f, "NONE", -3);
+		public static readonly Gas CarbonDioxide = new Gas("Carbon Dioxide", 30, 44.01f, false, 0.4f, "NONE", -3);
+
+		public static readonly Gas NitrousOxide = new Gas("Nitrous Oxide", 40, 44.01f, true, 0.4f, "NO2", -5);
+		public static readonly Gas Hydrogen = new Gas("Hydrogen", 30, 44.01f, false, 0.4f, "NONE", -3);
+		public static readonly Gas WaterVapor = new Gas("Water Vapor", 40, 44.01f, true, 0.4f, "WaterVapour", -9);
+		public static readonly Gas BZ = new Gas("BZ", 20, 44.01f, false, 0.4f, "NONE", -3);
+		public static readonly Gas Miasma = new Gas("Miasma", 20, 44.01f, true, 0.4f, "Miasma", -7);
```

Comment:
I don't think Miasma is really that good of the system/would be appropriate in something other than atmospherics?

Reply:
I just ported over all the TG gases, if we feel like its not needed we can remove. I think pet said that miasma can cause disease. We could also make dead bodies give it off after they have been left a while.